scripts/adjust_energy_totals.py

```diff
@@ -0,0 +1,372 @@
+
+
+    # Adjustments for Germany
+    # accoording to the Max Scenario also used in the Netzentwicklungsplan
+
+    # residential space heating
+    # basierend auf BMWi-Zielszenario, Energiereferenzprognose, Seite 261
+    # residential hot water
+    # basierend auf BMWi-Zielszenario, Energiereferenzprognose, Seite  QUELLE
+    # conversion from PJ to TWh and from Final energy demand (including ambient
+    # heat) to delivered energy using "Nutzungsgrade"
+    # Entwicklung der Energiemärkte – Energiereferenzprognose, page: 267 + 268
+    # https://www.prognos.com/uploads/tx_atwpubdb/140716_Langfassung_583_Seiten_Energiereferenzprognose_2014.pdf
+    # 96% or 128% (excl. ambient air)
+    energy_totals.loc[(energy_totals.index == 'DE'
+                       ), 'total residential space'] = 845 * 0.96 /60/60*1000
+    # 94% (no value given excl. ambient air)
+    energy_totals.loc[(energy_totals.index == 'DE'
+                       ), 'total residential water'] = 226 * 0.94 /60/60*1000
+    # service space heating
+    # basierend auf BMWi-Zielszenario, Energiereferenzprognose, Seite QUELLE
+    # service hot water
+    # basierend auf BMWi-Zielszenario, Energiereferenzprognose, Seite  QUELLE
+    # "Nutzungsgrad" not given, assumption: as for residential sector
+    energy_totals.loc[(energy_totals.index == 'DE'
+                       ), 'total services space'] = 151 * 0.96 /60/60*1000
+    # 94% (no value given excl. ambient air)
+    energy_totals.loc[(energy_totals.index == 'DE'
+                       ), 'total services water'] = 185 * 0.94 /60/60*1000
+
+
+    """
+    IMPORTANT INTERFACE REMARKS
+    'electricity residential space' and 'electricity residential water' and
+    'electricity services space' and 'electricity services water'
+    are adjusted at a different place. Where? Please add a cross reference.
+
+    The following columns are not updated and should not be used. Can we delete
+    them to be on the safe side?
+    'total residential cooking', 'electricity residential cooking'
+    'total residential', 'electricity residential'
+    'total services cooking', electricity services cooking',
+    'total services', 'electricity services'
+    """
+
+    return energy_totals
+
+
+def adjust_CO2_totals():
+    """
+    DO WE CORRECT THE EMISSIONS???
+
+    Returns
+    -------
+    None.
+
+    """
+
+    return None
+
+def adjust_district_heating_shares():
+    """
+    Adjust future district heating shares for the scenario generation
+
+    Adjust the district heating shares for European countries using hotmaps
+    data. The assumption for Germany is changed afterwards again.
+
+    Parameters
+    ----------
+        None
+
+    Returns
+    -------
+        None
+
+    ToDo
+    ----
+         Implementation: Load the of the csv file and change the data.
+
+    Notes
+    -----
+        None.
+
+    """
+    # implementation is still missing
+
+    return None
```

Comment:
The adjust_district_heating_shares() function can be deleted. The adjustment of the district heating shares was implemented in a different way.

Reply:
shall I do it @ClaraBuettner ?